src/Histogram.java

```diff
@@ -1,22 +1,65 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public class Histogram {
+/**
+ * Simple implementation for representing histograms. Variables are stored
+ * internally as doubles. Includes functions to read from basic (space/tab/
+ * comma delimited) format and json data sources.
+ * 
+ * Uses private constructors and static from* methods.
+ * 
+ * @author Andrew Gerber
+ */
+public class Histogram implements ProbabilityDistribution {
```

Comment:
Perhaps name this class `HistogramDistribution` or `ExplicitDistribution`? The name `Histogram` is slightly vague.